.github/workflows/ci.yml

```
@@ -19,7 +19,7 @@ jobs:
 
       - name: Fetch path to Zcash parameters
         working-directory: ./zcash_proofs
```

I think this is needed to get Windows working:
```suggestion
        working-directory: ./zcash_proofs
        shell: bash
```